autogluon/utils/tabular/ml/models/fastainn/hyperparameters/parameters.py

```diff
@@ -0,0 +1,48 @@
+from .....ml.constants import BINARY, MULTICLASS, REGRESSION
+
+
+def get_param_baseline(problem_type, num_classes=None):
```

Comment:
Can you add a TODO comment here to generalize some of these functions somehow? Seems there's an opportunity for code reduction with some other models

Reply:
Added a comment for `get_param_baseline()`; other methods and hyper-parameters are model-specific and not reusable.